src/mlpack/methods/ann/layer/sequential.hpp

```diff
@@ -87,6 +88,12 @@ class Sequential
    */
   Sequential(const bool model, const bool ownsLayers);
 
+  //! Copy constructor.
+  Sequential(const Sequential&);
```

Comment:
The implementation is missing:

https://github.com/mlpack/mlpack/pull/2414/commits/1dddedfbe268dabebe413b26839c52d2ea00ac4b#diff-a063bb375e9dcb916e30602027ff877e

Reply:
I tried all the above, none seems to work

Reply:
I see, so I delete the copy constructor here.